src/App.tsx

```diff
@@ -1,14 +1,78 @@
-import "./styles/App.css";
-import Header from "./components/Header";
-import Intro from "./pages/Intro";
+import React from "react";
+import ReactDOM from "react-dom/client";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { RouterProvider, createBrowserRouter } from "react-router-dom";
+import ErrorPage from "./pages/Error.tsx";
+import ClientLayout from "./pages/layouts/ClientLayout.tsx";
+import Home from "./pages/client/Home.tsx";
+import Explore from "./pages/client/Explore.tsx";
+import Message from "./pages/client/Message.tsx";
+import IntroLayout from "./pages/layouts/IntroLayout.tsx";
+import FormCreateUser from "./components/forms/createUserForm.tsx";
+import "./styles/main.css";
+import UserList from "./pages/admin/UserList.tsx";
+import AdminLayout from "./pages/layouts/AdminLayout.tsx";
+import PostList from "./pages/admin/PostList.tsx";
+import CommentList from "./pages/admin/CommentList.tsx";
 
-function App() {
-  return (
-    <>
-      <Header />
-      <Intro />
-    </>
-  );
-}
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <IntroLayout />,
+    errorElement: <ErrorPage />,
+  },
+  {
+    path: "/",
+    element: <ClientLayout />,
+    children: [
+      {
+        path: "home",
```

Comment:
Please use: src/constants/routes.ts for these routes

Reply:
I have fixed it.